src/main/java/com/ombremoon/enderring/common/init/entity/StatusEffectInit.java

```diff
@@ -154,7 +155,7 @@ public class StatusEffectInit {
                     .addAttribute(new AttributeModifier(
                                     UUID.fromString("d8b1d3c-04cc-64b3-6b08-67b8bdb40834"),
                                     "MAGICSHROUDING_CRACKED",
-                                    0.7F, AttributeModifier.Operation.ADDITION),
+                                    7F, AttributeModifier.Operation.ADDITION),
```

Comment:
This has changed so when increasing damage the tear should have no attribute modifier and the damage increase should be added in DamageUtil#getApMultipliers